### ISSUE TITLE
What helpdesk software do you use that puts a little text baloon a the bottom of your pages. And then brings up this handy little window?  Thanks, JIm.

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#3595]
**What helpdesk software do you use that puts a little text baloon a the bottom of your pages. And then brings up this handy little window? Thanks, JIm.**

-----

User: [See in registry](https://www.gbif.org/api/feedback/user/663b9869850608e5b7e3ea07e5dceff0:57ea10cac5eeec608adb6204645e3b4d1dac025c9e4b08b0d22cdd8fb4deeda6720beb7c5549b0e92cf598ef9c0a9b18c7f6d3efdf76bb4185dacce44624323c)
System: Firefox 92.0.0 / Windows 10.0.0
Referer: https://www.gbif.org/occurrence/656953475
Window size: width 1413 - height 1111
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2021-09-16T02:01:31.732Z',to:'2021-09-16T02:07:31.732Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2021-09-16T02:01:31.732Z',to:'2021-09-16T02:07:31.732Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: INFO
datasetKey: 8f79c802-a58c-447f-99aa-1d6a0790825a
publishingOrgKey: b554c320-0560-11d8-b851-b8a03c50a862
Network keys: 99d66b6c-9087-452f-a9d4-f15f2c2d0e7e 
Node handles: @albenson-usgs

**Comment by MortenHofft** [comment]
Hi Jim
I'm happy to hear that you find it useful. We do not have any dedicated helpdesk software. 

We used to have a self hosted JIRA instance, but that was abandoned in an effort to reduce the number of systems we maintain that isn't directly related to biodiversity.

What we do now is to have people fill in a form and post that content to Github (which is where all our code lives). The balloon and the form is something we wrote ourselves. 

It works as intended but it isn't ideal either. 

**Negative**
* Answering users is kind of awkward for example. I do not want to expose your email address, but I want to contact you. So we encode the email in a link so that only staff in the secretariat can see read it. And answering users (that haven't linked a Github account) require me to send an email, which kind of breaks the link to this issue and the help desk flow.
* And all issues are public, at times it would be nice to have the option for private issues with individual users.
* We also get email with help desk questions, so we really have two channels to monitor, it would be nicer with just one.
* To begin with we exposed number of open issues related to e.g. an occurrences on the individual occurrences pages, but that required too many calls to the Github API and we were blocked.
* Users do not have an easy way to see all the issues they have left and they are not updated on activities.

**Postive**
* On the positive side then we do not have to maintain (or pay) for any additional software.
* And we can easily move issues to the repositories where the code they relate to live.
* It is fairly simple and flexible.

> user informed per email
